packages/calcite-components/src/components/alert/alert.tsx

```diff
@@ -460,24 +470,47 @@ export class Alert implements OpenCloseComponent, LoadableComponent, T9nComponen
 
   private lastMouseOverBegin: number;
 
-  private queueTimeout: number;
+  private queueTimeoutId: number = null;
 
   private totalOpenTime = 0;
 
   private totalHoverTime = 0;
 
+  private isHovered: boolean;
+
   openTransitionProp = "opacity";
 
   transitionEl: HTMLDivElement;
 
-  mouseFocus: boolean;
```

Comment:
No such thing as a 🐭 focus.

Reply:
_Detective Pikachu steps into the chat_

![detective-pikachu-magnifying-glass](https://github.com/user-attachments/assets/3ec8ebdf-ef21-4c21-a9a6-961d1597c0d6)